HedvigInsurance/web-onboarding
# GRW-697 - Show error modal if credit check fails

## What?

Show error modal at checkout when backend returns a 403 with errorCode "MANUAL_REVIEW_REQUIRED"

## Why?

We need to alert users that they need to get in touch with our customer service teams

## How to test

* Need to get in touch with @fredriklagerblad to turn on the credit check on staging
* He or I can provide and NO SSN that generates this error

**Ticket(s): [GRW-697]**

## Screenshots / recordings 

https://user-images.githubusercontent.com/89857278/151935305-91301f9c-60d7-4ebc-a268-aaef1a475116.mov


[GRW-697]: https://hedvig.atlassian.net/browse/GRW-697?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
